salt/modules/rabbitmq.py

```diff
@@ -111,15 +117,14 @@ def _get_rabbitmq_plugin():
     This works by taking the rabbitmq-server version and looking for where it
     seems to be hidden in /usr/lib.
     '''
-    rabbitmq = __context__.get('rabbitmq-plugins')
+    global RABBITMQ_PLUGINS
 
-    if rabbitmq is None:
+    if RABBITMQ_PLUGINS is None:
         version = __salt__['pkg.version']('rabbitmq-server').split('-')[0]
-        rabbitmq = ('/usr/lib/rabbitmq/lib/rabbitmq_server-{0}'
-                    '/sbin/rabbitmq-plugins').format(version)
-        __context__['rabbitmq-plugins'] = rabbitmq
+        RABBITMQ_PLUGINS = ('/usr/lib/rabbitmq/lib/rabbitmq_server-{0}'
+                            '/sbin/rabbitmq-plugins').format(version)
```

Comment:
There are reasonable points on both sides here but I am going to side with @skizunov and merge this.

@scott-w thanks for coming by and I'm glad you participated in this discussion and for your work on this module. It's very much appreciated. :]